chain/chunks/README.md

```diff
@@ -0,0 +1,30 @@
+# near-chunks
+
+This crate cotains functions to handle chunks. In NEAR - the block consists of multiple chunks - at most one per shard.
+
+When chunk is created, the creator encodes it using Reed Solomon encoding (ErasureCoding) - creating PartialChunks, that are later sent to all the validators (each validator gets a subset of them).
```

Comment:
I think it would also be helpful to mention that `PartialEncodedChunks` is comprised of parts and receipts. A part denotes a part from the Reed Solomon encoding and they can be used to reconstruct the full chunk if a node receives enough parts. Cross-shard receipts are also sent as part of `PartialEncodedChunks`